### PR TITLE
[EN/RU] Update `COT/COT2022` for the Qualifiers

### PR DESCRIPTION
**Pull request by Kimitakari**
Also fixed minor fixes

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by TicClick — APPROVED** [review]
нормально

File: wiki/Tournaments/COT/2022/en.md [hunk 1]
```diff
@@ -53,9 +53,26 @@ COT 2022 is run by various osu! community members mostly hailing from the Common
...
+2. [BESTie - Excuse Me (Euny) \[ShadrEuny's Ex\]](https://osu.ppy.sh/beatmapsets/423925#fruits/915925)
+- Hidden
+1. [A-One feat. Shihori - Magic Girl !! (Greaper) \[Deif's Rain\]](https://osu.ppy.sh/beatmapsets/1644488#fruits/3356804)
+2. [Jenny Rom - The Game of Love (E=MC2) Mix (ztrot) \[Insane\]](https://osu.ppy.sh/beatmapsets/14412#fruits/52741)
+- HardRock
+1. [Lime - Chronomia (Rocma) \[Rain\]](https://osu.ppy.sh/beatmapsets/1471328#fruits/3207951)
+2. [Helblinde - Above the Clouds ([ Rizen ]) \[Timeless\]](https://osu.ppy.sh/beatmapsets/344252#fruits/785162)
+- DoubleTime
+1. [ZUTOMAYO - Humanoid (Jemzuu) \[Imai Lisa & Du5t's Platter\]](https://osu.ppy.sh/beatmapsets/1052572#fruits/2241108)
+2. [XX:me - Torikago (Mordred) \[Dakini's Insane\]](https://osu.ppy.sh/beatmapsets/764067#fruits/1624401)
```
Comment (Bazzadwarf) [inline wiki/Tournaments/COT/2022/en.md:74]:
```suggestion
- NoMod
  1. [Koinata Miho, Jougasaki Mika, Hayami Kanade - Twintail no Kaze (Spectator) \[Collab Overdose\]](https://osu.ppy.sh/beatmapsets/1328225#fruits/2751483)
  2. [BESTie - Excuse Me (Euny) \[ShadrEuny's Ex\]](https://osu.ppy.sh/beatmapsets/423925#fruits/915925)
- Hidden
  1. [A-One feat. Shihori - Magic Girl !! (Greaper) \[Deif's Rain\]](https://osu.ppy.sh/beatmapsets/1644488#fruits/3356804)
  2. [Jenny Rom - The Game of Love (E=MC2) Mix (ztrot) \[Insane\]](https://osu.ppy.sh/beatmapsets/14412#fruits/52741)
- HardRock
  1. [Lime - Chronomia (Rocma) \[Rain\]](https://osu.ppy.sh/beatmapsets/1471328#fruits/3207951)
  2. [Helblinde - Above the Clouds ([ Rizen ]) \[Timeless\]](https://osu.ppy.sh/beatmapsets/344252#fruits/785162)
- DoubleTime
  1. [ZUTOMAYO - Humanoid (Jemzuu) \[Imai Lisa & Du5t's Platter\]](https://osu.ppy.sh/beatmapsets/1052572#fruits/2241108)
  2. [XX:me - Torikago (Mordred) \[Dakini's Insane\]](https://osu.ppy.sh/beatmapsets/764067#fruits/1624401)
```

File: wiki/Tournaments/COT/2022/ru.md [hunk 2]
```diff
@@ -59,6 +59,23 @@ COT 2022 организован различными членами сообще
...
+2. [BESTie - Excuse Me (Euny) \[ShadrEuny's Ex\]](https://osu.ppy.sh/beatmapsets/423925#fruits/915925)
+- Hidden
+1. [A-One feat. Shihori - Magic Girl !! (Greaper) \[Deif's Rain\]](https://osu.ppy.sh/beatmapsets/1644488#fruits/3356804)
+2. [Jenny Rom - The Game of Love (E=MC2) Mix (ztrot) \[Insane\]](https://osu.ppy.sh/beatmapsets/14412#fruits/52741)
+- HardRock
+1. [Lime - Chronomia (Rocma) \[Rain\]](https://osu.ppy.sh/beatmapsets/1471328#fruits/3207951)
+2. [Helblinde - Above the Clouds ([ Rizen ]) \[Timeless\]](https://osu.ppy.sh/beatmapsets/344252#fruits/785162)
+- DoubleTime
+1. [ZUTOMAYO - Humanoid (Jemzuu) \[Imai Lisa & Du5t's Platter\]](https://osu.ppy.sh/beatmapsets/1052572#fruits/2241108)
+2. [XX:me - Torikago (Mordred) \[Dakini's Insane\]](https://osu.ppy.sh/beatmapsets/764067#fruits/1624401)
```
Comment (Bazzadwarf) [inline wiki/Tournaments/COT/2022/ru.md:77]:
```suggestion
- NoMod
  1. [Koinata Miho, Jougasaki Mika, Hayami Kanade - Twintail no Kaze (Spectator) \[Collab Overdose\]](https://osu.ppy.sh/beatmapsets/1328225#fruits/2751483)
  2. [BESTie - Excuse Me (Euny) \[ShadrEuny's Ex\]](https://osu.ppy.sh/beatmapsets/423925#fruits/915925)
- Hidden
  1. [A-One feat. Shihori - Magic Girl !! (Greaper) \[Deif's Rain\]](https://osu.ppy.sh/beatmapsets/1644488#fruits/3356804)
  2. [Jenny Rom - The Game of Love (E=MC2) Mix (ztrot) \[Insane\]](https://osu.ppy.sh/beatmapsets/14412#fruits/52741)
- HardRock
  1. [Lime - Chronomia (Rocma) \[Rain\]](https://osu.ppy.sh/beatmapsets/1471328#fruits/3207951)
  2. [Helblinde - Above the Clouds ([ Rizen ]) \[Timeless\]](https://osu.ppy.sh/beatmapsets/344252#fruits/785162)
- DoubleTime
  1. [ZUTOMAYO - Humanoid (Jemzuu) \[Imai Lisa & Du5t's Platter\]](https://osu.ppy.sh/beatmapsets/1052572#fruits/2241108)
  2. [XX:me - Torikago (Mordred) \[Dakini's Insane\]](https://osu.ppy.sh/beatmapsets/764067#fruits/1624401)
```

**Review by Bazzadwarf — CHANGES_REQUESTED** [review]
Fixes failing check due to incorrect list indentation.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**